### PR TITLE
Igor/epd 1141 implement the actual script to generate

### PR DESCRIPTION
**Pull request by IgorHorta**
Goal here is to implement a simple page to allow users to download a PDF containing all the possible questions in a formsort form

https://github.com/vial-technology/formsort-pdf-form-generator/assets/6900853/4a6c1524-1596-4462-acb0-8b7a917f390c

### CODE REVIEW

File: answers-pdf-service/src/env.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { z } from 'zod';
 
 const EnvSettings = z.object({
-  AWS_REGION: z.string().min(1),
-  AWS_BUCKET_NAME: z.string().min(1),
+  // AWS_REGION: z.string().min(1),
```
Comment (zaindewsi) [inline answers-pdf-service/src/env.ts]: do we need this line here?

File: answers-pdf-service/src/flow-content-api.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ export const getFlowContentJSONFromAPI = async (
     headers: { 'x-api-key': FORMSORT_API_KEY },
   });
   const json = await response.json();
+  console.log({json: JSON.stringify(json)})
```
Comment (zaindewsi) [inline answers-pdf-service/src/flow-content-api.ts]:
```suggestion

```

**Review by zaindewsi — APPROVED** [review]

File: answers-pdf-service/src/env.ts (hunk 1 above)
Comment (IgorHorta) [reply answers-pdf-service/src/env.ts]: totally forgot! will fix it

File: answers-pdf-service/src/flow-content-api.ts (hunk 2 above)
Comment (IgorHorta) [reply answers-pdf-service/src/flow-content-api.ts]: thankss

**Pull request merged**